account_reconciliation_partner_placeholder/static/src/js/reconciliation/reconciliation_renderer.js

```diff
@@ -0,0 +1,21 @@
+odoo.define('lr_account.ReconciliationRenderer', function (require) {
```

Comment:
Would it be good convention to use the full module name for the prefix?

Reply:
Yeah, I think so. We were using that as a 'de facto' convention.